test/system/helpers.bash

```diff
@@ -847,6 +847,21 @@ function _ensure_container_running() {
     die "Timed out waiting for container $1 to enter state running=$2"
 }
 
+# Return the config digest of an image in containers-storage.
+# The input can be a named reference, or an @imageID (including shorter imageID prefixes)
+# Historically, the image ID was a good indicator of “the same” image;
+# with zstd:chunked, the same image might have different IDs depending on whether
+# creating layers happened based on the TOC (and per-file operations) or the full layer tarball
+function image_config_digest() {
+    echo "image_config_digest $1" >&2
```

Comment:
Did you intend to leave these debug statements?

The `printf | sha256sum` below is subtle and potentially distracting; if you remove the debug statements, you could collapse everything down to
```bash
    local sha_output; sha_output=$(skopeo inspect --raw --config "containers-storage:$1" | sha256sum)
    # strip filename ("-") from sha output
    # echo "${sha_output%% *}"
```
I'm fine with leaving the debug statements in place for a few months, but if so please add a comment explaining when it might be safe to remove them, and explaining the reason for `printf` (preserving exact output without the additional newline that `<<<"$config"` would introduce)

Reply:
After the `local` mystery of https://github.com/containers/podman/pull/24286#discussion_r1805491648 , the most useful case of detecting broken inputs should be handled naturally; so I have dropped the logging.